mentorship iosTests/TaskCommentsTests.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  TaskCommentsTests.swift
+//  Created on 30/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import XCTest
+@testable import mentorship_ios
+
+class TaskCommentsTests: XCTestCase {
+    // custom urlsession for mock network calls
+    var urlSession: URLSession!
+
+    override func setUpWithError() throws {
+        // Set url session for mock networking
+        let configuration = URLSessionConfiguration.ephemeral
+        configuration.protocolClasses = [MockURLProtocol.self]
+        urlSession = URLSession(configuration: configuration)
+    }
+
+    override func tearDownWithError() throws {
+        urlSession = nil
+        super.tearDown()
+    }
+
+    // MARK: - Service Tests
+    
+    func testFetchComments() throws {
+        let taskCommentsService: TaskCommentsService = TaskCommentsAPI(urlSession: urlSession)
+        
+        // Set mock json and data
+        let mockJSON: [TaskCommentsModel.TaskCommentsResponse] = [
```

Comment:
nit: mockJSON data for every test could go into a separate JSON file, and you could read that file in different tests to reusue the same data. Since most of the data would be related to the different Models we have, reuse is definitely possible.

Reply:
Understood. Can I do this over the weekend, seems like a tedious task 😅